hack/bazel-test-affected.sh

```diff
@@ -32,26 +32,27 @@ set -o nounset
 set -o pipefail
 
 # Compute list of modified files in bazel package form.
-packages=($(bazel query \
+mapfile -t packages < <(bazel query \
```

Comment:
IIRC kubernetes has tried to make this fix before, but stopped because this doesn't work on macos

Reply:
`mapfile` that is. cc @cblecker who I seem to remember being involved

Reply:
Can confirm. The bash included with darwin (3.2) doesn't support mapfile (4.x).

Reply:
I can convert this to read -a